pyproject.toml

```diff
@@ -1,52 +1,227 @@
 [build-system]
-requires = ["setuptools >= 50", "wheel", "setuptools_scm[toml] >= 6"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
```

Comment:
I would rather not change the build backend mixed with other things. Make it its own PR.

Reply:
Moved back to setuptools.